skrub/datasets/_ken_embeddings.py

```diff
@@ -177,6 +178,39 @@ def get_ken_embeddings(
     For searching the types, the :func:`~skrub.datasets.get_ken_types`
     function can be used.
 
+    Examples
+    --------
+    :func:`get_ken_embeddings` allows you to extract embeddings
+    you are interested in. For instance, if we are interested in
+    video games:
+
+    >>> games_embedding = get_ken_embeddings(types="video_games")
+    >>> games_embedding.head()
+                            Entity                                       Type  ...      X198      X199
+    0       <The_Mysterious_Island>  <wikicat_novels_adapted_into_video_games>  ... -0.072814 -0.156973
```

Comment:
I don't understand: we still have the "<" here, it seems

Reply:
Thanks, corrected, after I merged with #601